test/base.service.test.js

```diff
@@ -0,0 +1,77 @@
+require('../src/util/dbconfig');
+const testMeasure = require('./fixtures/testMeasure.json');
+const testLibrary = require('./fixtures/testLibrary.json');
+const testPatient = require('./fixtures/testPatient.json');
+const { testSetup, cleanUpDb } = require('./populateTestData');
+const supertest = require('supertest');
+const { buildConfig } = require('../src/util/config');
+const { initialize } = require('../src/server/server');
+const config = buildConfig();
+const server = initialize(config);
+const updatePatient = { id: 'testPatient', name: 'anUpdate' };
+describe('base.service', () => {
+  beforeAll(async () => {
+    await testSetup(testMeasure, testPatient, testLibrary);
+  });
+
+  describe('create', () => {
+    test('test create with correct headers', async () => {
+      await supertest(server.app)
+        .post('/4_0_0/Patient')
+        .send(testPatient)
+        .set('Accept', 'application/json+fhir')
+        .set('content-type', 'application/json+fhir')
+        .expect(201)
+        .then(async response => {
+          // Check the response
+          expect(response.body._id).not.toBeNull();
+          expect(response.body._id).not.toBe(testMeasure.id);
+          expect(response.headers.location).not.toBeNull();
```

Comment:
```suggestion
          expect(response.headers.location).toBeDefined();
```

The first 2 assertions don't really mean anything. Mongo is always gonna create a `_id` and it will never match. It's just part of mogno no need to test that with these.

Suggest making the assertion here `.toBeDefined()`, I think if it's undefined, that would still pass this assertion even though that's undesired